she/verify.py

```
@@ -0,0 +1,239 @@
+Converts dual gates, domain tests, security checks, and regression invariants
+into an explicit evidence contract for an incident. This module plans checks;
+it does not run Actions, mutate git, or call the network.
+
+Live execution remains a later slice. Results may be recorded later via
+`apply_check_results` without changing the planned gate set.
+stdlib-only.
+"""
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Any, Mapping
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)